include/sound/soc-acpi.h

```diff
@@ -94,6 +114,7 @@ struct snd_soc_acpi_mach_params {
 struct snd_soc_acpi_mach {
 	const u8 id[ACPI_ID_LEN];
 	const u32 link_mask;
+	const struct snd_soc_acpi_link *links;
```

Comment:
need to split this commit in two
1. header file definition
2. acpi table changes